<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin</class>
 <widget class="QMainWindow" name="admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员界面</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">background-image: url(C:/Users/wxy/Desktop/123.jpg);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>581</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>440</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover
{
	background-color: qlineargradient(spread:pad, x1:0.909, y1:0.864, x2:0.926, y2:0, stop:0.448864 rgba(255, 239, 88, 250), stop:1 rgba(255, 255, 255, 255));
	
}
</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_show">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover
{
	background-color: qlineargradient(spread:pad, x1:0.909, y1:0.864, x2:0.926, y2:0, stop:0.448864 rgba(255, 239, 88, 250), stop:1 rgba(255, 255, 255, 255));
	
}
</string>
    </property>
    <property name="text">
     <string>学生信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_change">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>添加信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_del">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>删除信息</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_findname">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>学号</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>年龄</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>学分</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>成绩</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>课程号</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_order">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover
{
	background-color: qlineargradient(spread:pad, x1:0.909, y1:0.864, x2:0.926, y2:0, stop:0.448864 rgba(255, 239, 88, 250), stop:1 rgba(255, 255, 255, 255));
	
}
</string>
    </property>
    <property name="text">
     <string>排序</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_desc">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>降序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_show_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover
{
	background-color: qlineargradient(spread:pad, x1:0.909, y1:0.864, x2:0.926, y2:0, stop:0.448864 rgba(255, 239, 88, 250), stop:1 rgba(255, 255, 255, 255));
	
}
</string>
    </property>
    <property name="text">
     <string>课程信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_show_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover
{
	background-color: qlineargradient(spread:pad, x1:0.909, y1:0.864, x2:0.926, y2:0, stop:0.448864 rgba(255, 239, 88, 250), stop:1 rgba(255, 255, 255, 255));
	
}
</string>
    </property>
    <property name="text">
     <string>学生成绩</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>姓名搜索</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>学号搜索</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>课程搜索</string>
     </property>
    </item>
   </widget>
   <widget class="QTableView" name="tableView_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>老师</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_show_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>修改密码</string>
    </property>
   </widget>
   <zorder>label_4</zorder>
   <zorder>tableView</zorder>
   <zorder>pushButton_exit</zorder>
   <zorder>pushButton_show</zorder>
   <zorder>pushButton_change</zorder>
   <zorder>pushButton_del</zorder>
   <zorder>lineEdit_findname</zorder>
   <zorder>comboBox</zorder>
   <zorder>pushButton_order</zorder>
   <zorder>checkBox_desc</zorder>
   <zorder>pushButton_show_2</zorder>
   <zorder>pushButton_show_3</zorder>
   <zorder>comboBox_2</zorder>
   <zorder>tableView_2</zorder>
   <zorder>label</zorder>
   <zorder>pushButton_show_4</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>732</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
